simulatingrisk/risky_bet/model.py

```diff
@@ -221,15 +221,16 @@ def risk_max(self):
 
     @property
     def risk_q1(self):
-        if self.agent_risk_levels:
-            risk_median = self.risk_median
-            # first quartile is the median of values less than the median
-            return statistics.median(
-                [r for r in self.agent_risk_levels if r < risk_median]
-            )
+        risk_median = self.risk_median
```

Comment:
Cool way to get a quartile as a median of medians. I use and often see people use numpy for this purpose: https://numpy.org/doc/stable/reference/generated/numpy.quantile.html

Reply:
I looked at the python built-in statistics quantiles method; can't remember if I looked at numpy - do you know if there's a significant difference between them?  (I don't remember why I didn't go with the quantiles method)

that said, I'm not sure I'm actually using these methods anymore for data collection/analysis - and if I'm not, I should remove the code - it would be easier to use analysis libraries like numpy on the collected data after the fact